### DISCUSSION TITLE
Prime ng Ultima with latest prime ng version (16.9.1)

### DISCUSSION THREAD

**Discussion by hiteshlinge** [primefaces/community#627] [PrimeNG] [answered]
hi,

I am using the ultima primeng template for my application, now I want to update my application to latest prime ng version. But current ultima application is using prime ng 16.0.2, when can we expect the ultima with prime ng 16.9.1 ?

Just FYI, I tried updating the primeng version of my package.json to 16.9.2, but some how my application gets hangs after launch

Thanks

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Ultima will not receive 16.9.1 update, it'll receive v17 update. You can simply update your primeng version and related angular versions to the 16.9.1

**Comment by AshishFreak** [comment]
hi can you share theme i cant purchase from store
